src/einsteinpy/tests/test_plotting/test_staticgeodesicplotter.py

```diff
@@ -54,3 +67,121 @@ def test_plot_save_saves_plot(mock_save):
     name = "test_plot.png"
     cl.save(name)
     mock_save.assert_called_with(name)
+
+
+@mock.patch("einsteinpy.plotting.geodesics_static.StaticGeodesicPlotter.plot_attractor")
+def test_plot_calls_get_x_y2(mock_plot_attractor):
+    """
+    Test for checking the colors passed
+    """
+    r = [306 * u.m, np.pi / 2 * u.rad, np.pi / 2 * u.rad]
+    v = [0 * u.m / u.s, 0 * u.rad / u.s, 951.0 * u.rad / u.s]
+    m = 4e24 * u.kg
+    el = 0.002
+    ss = 0.5e-6
+    cl = StaticGeodesicPlotter(m, attractor_color="blue")
+    cl.plot(r, v, el, ss)
+    mock_plot_attractor.assert_called_with()
+    assert cl._attractor_present
+    assert cl.attractor_color == "blue"
+
+
+@mock.patch("einsteinpy.plotting.geodesics_static.StaticGeodesicPlotter.get_swcradius")
+def test_plot_get_swcradius(mock_get_swcradius):
+    """
+    Test for get_swcradius()
+    """
+    r = [306 * u.m, np.pi / 2 * u.rad, np.pi / 2 * u.rad]
+    v = [0 * u.m / u.s, 0 * u.rad / u.s, 951.0 * u.rad / u.s]
+    m = 4e24 * u.kg
+    el = 0.002
+    ss = 0.5e-6
+    cl = StaticGeodesicPlotter(m)
+    cl.plot(r, v, el, ss)
+    cl.get_swcradius()
+    mock_get_swcradius.assert_called_with()
+
+
+@mock.patch("einsteinpy.plotting.geodesics_static.plt.show")
+def test_plot_show_shows_plot(mock_show):
+    """
+    Test for show()
+    """
+    r = [306 * u.m, np.pi / 2 * u.rad, np.pi / 2 * u.rad]
+    v = [0 * u.m / u.s, 0 * u.rad / u.s, 951.0 * u.rad / u.s]
+    m = 4e24 * u.kg
+    el = 0.002
+    ss = 0.5e-6
+    cl = StaticGeodesicPlotter(m)
+    cl.plot(r, v, el, ss)
+    cl.show()
+    mock_show.assert_called_with()
+
+
+@mock.patch("einsteinpy.plotting.geodesics_static.StaticGeodesicPlotter.plot_attractor")
+def test_plot_calls_plot_attractor2(mock_plot_attractor):
+    """
+    Test for checking the attractor_radius_scale passed
+    """
+    r = [306 * u.m, np.pi / 2 * u.rad, np.pi / 2 * u.rad]
+    v = [0 * u.m / u.s, 0 * u.rad / u.s, 951.0 * u.rad / u.s]
+    m = 4e24 * u.kg
+    el = 0.002
+    ss = 0.5e-6
+    cl = StaticGeodesicPlotter(m, attractor_radius_scale=1500)
+    cl.plot(r, v, el, ss)
+    mock_plot_attractor.assert_called_with()
+    assert cl._attractor_present
+    assert cl.attractor_radius_scale == 1500
+
+
+@mock.patch("einsteinpy.plotting.geodesics_static.StaticGeodesicPlotter.plot_attractor")
+def test_plot_calls_plot_attractor3(mock_plot_attractor):
```

Comment:
I see you have written same test multiple times. Why?

Reply:
Multiple tests are for different functions made in the staticgeodesicplotter. Just named them 1,2,3 in hurry to see if it passes the checks